src/jvm/test/resources/expected_logs/validation_function_failure.txt

```diff
@@ -18,24 +18,24 @@ All operations above the horizontal line | ----- | happen before those below the
 
 java.lang.IllegalStateException: Validation works!
 	at org.jetbrains.kotlinx.lincheck_test.representation.ValidationFunctionCallTest.validateWithError(ValidationFunctionTests.kt:41)
-	at org.jetbrains.kotlinx.lincheck.runner.TestThreadExecution1421.run(Unknown Source)
-	at org.jetbrains.kotlinx.lincheck.runner.FixedActiveThreadsExecutor.testThreadRunnable$lambda$10(FixedActiveThreadsExecutor.kt:172)
-	at java.lang.Thread.run(Thread.java:750)
+	at org.jetbrains.kotlinx.lincheck.runner.TestThreadExecution9.run(Unknown Source)
+	at org.jetbrains.kotlinx.lincheck.runner.FixedActiveThreadsExecutor.testThreadRunnable$lambda$10(FixedActiveThreadsExecutor.kt:171)
+	at java.base/java.lang.Thread.run(Thread.java:840)
 
 
 The following interleaving leads to the error:
-| -------------------------------------------------------- |
-|                  Thread 1                  |  Thread 2   |
-| -------------------------------------------------------- |
-| operation()                                |             |
-| -------------------------------------------------------- |
-|                                            | operation() |
-| operation()                                |             |
-| -------------------------------------------------------- |
-| operation()                                |             |
-| -------------------------------------------------------- |
-| validateWithError(): IllegalStateException |             |
-| -------------------------------------------------------- |
+| ----------------------------------------------------------- |
+|                   Thread 1                    |  Thread 2   |
+| ----------------------------------------------------------- |
+| operation()                                   |             |
+| ----------------------------------------------------------- |
+|                                               | operation() |
+| operation()                                   |             |
+| ----------------------------------------------------------- |
+| operation()                                   |             |
+| ----------------------------------------------------------- |
+| validateWithError(): IllegalStateException #1 |             |
```

Comment:
Should we print exception number near validation exception stack trace (similarly as we do for other types of exceptions) ?

Reply:
commit `Add number to validation function stack trace.` solves this issue. However, I don't really like the solution. Also, in GPMC mode we have the same problem look at `access_field_representation_test.txt`